Sources/KSCrashRecording/Monitors/KSCrashMonitor_CPPException.cpp

```diff
@@ -145,34 +159,37 @@ static void CPPExceptionTerminate(void)
         memset(crashContext, 0, sizeof(*crashContext));
 
         char descriptionBuff[DESCRIPTION_BUFFER_LENGTH];
```

Comment:
I think we should pre-init this as `{0}` to avoid any possible surprises later on.
```suggestion
        char descriptionBuff[DESCRIPTION_BUFFER_LENGTH] = {0};
```

Reply:
ok. done.